README.md

```diff
@@ -35,24 +35,54 @@ To set up, just clone the Aladdin GitHub repository to get started:
 
     $ git clone git@github.com:fivestars-os/aladdin.git
     $ cd aladdin
-    $ ./scripts/infra_k8s_check.sh
+    $ scripts/infra_k8s_check.sh
 
-The `infra_k8s_check.sh` script checks to see if all of aladdin's dependencies are installed. Depending on the dependency, it will warn if it is missing, or install it in ~/.aladdin/bin if possible. This script is also run every time you run the aladdin.
+The `infra_k8s_check.sh` script checks to see if all of aladdin's dependencies are installed. Depending on the dependency, it will warn if it is missing or install it in `~/.aladdin/bin` if possible. This script is also run every time you run aladdin.
 
 You may want to add aladdin to your path:
 
-    $ eval $(./aladdin.sh env) 
-    
+    $ eval $(./aladdin.sh env)
+
 You may also want to add `~/.aladdin/bin` to your path:
 
     $ export PATH=$PATH:~/.aladdin/bin
 
 You can add these two commands to your profile file if desired.
 
-You will now need to [create your aladdin configuration](./docs/create_aladdin_configuration.md), and link that to aladdin. 
+You will now need to [create your aladdin configuration](./docs/create_aladdin_configuration.md), and link that to aladdin.
 
     $ aladdin config set config_dir /path/to/aladdin/configuration
 
+### NFS mounts
+
+Aladdin uses minikube for local development which is a VirtualBox-based VM. Internally it mounts your `/Users` (or `/cygdrive/c/Users` for Cygwin users) directory to the root of the minikube filesystem. We have discovered that the `vboxsf` mounts are not as performant as NFS mounts and furthermore are not as reliable when it comes to detecting file changes made from the host. To address this, we replace the `vboxsf` mounts within minikube with NFS mounts. However, this requires a bit of manual setup on the host machine before it will take effect.
+
+#### NFS Host Setup
+
+1. Install an NFS service on your host
+    1. **macOS** You should already have it available.
```

Comment:
Is this a reasonable expectation for all macOS users? Or am I making an assumption that only holds true at Fivestars where we provision machines with certain settings?

Reply:
I think we're good here